scripts/travis-ci/run-test.sh

```diff
@@ -13,6 +13,9 @@ case "$1" in
     *)
         ln -s $MODULE_DIR $DRUPAL_DIR/modules/og
         cd $DRUPAL_DIR
+        git clone --branch 8.x-1.x https://git.drupal.org/project/composer_manager.git modules/composer_manager
```

Comment:
I asked @webflo and he says that for now this legit. We can also set up a composer.json file for travis but this will take more time.